packages/eslint-plugin/src/rules/init-declarations.ts

```diff
@@ -24,7 +24,12 @@ export default createRule<Options, MessageIds>({
       extendsBaseRule: true,
     },
     schema: baseRule.meta.schema,
-    messages: baseRule.meta.messages,
+    messages: baseRule.meta.messages ?? {
+      initialized:
+        "Variable '{{idName}}' should be initialized on declaration.",
+      notInitialized:
+        "Variable '{{idName}}' should not be initialized on declaration.",
+    },
```

Comment:
this rule has been using message ids since 5.12.
I guess we should keep this alive until we drop v5.0.0 from the supported ranges.